test/truefi2/lines-of-credit/CreditModel.test.ts

```diff
@@ -387,7 +432,7 @@ describe('CreditModel', () => {
       ].map(([collateral, ltvRatio, result]) =>
         it(`when ${collateral} TRU is staked with ltvRatio=${ltvRatio}%, borrow limit rises by up to $${result}`, async () => {
           await creditModel.setStakingConfig(ltvRatio * 100, 0)
-          expect(await creditModel.conservativeCollateralValue(mockPool.address, parseTRU(collateral))).to.equal(parseEth(result))
+          expect(await creditModel.conservativeCollateralValue(parseTRU(collateral))).to.equal(parseEth(result))
```

Comment:
Add tests to check how oracle price affects `conservativeCollateralValue` `conservativeCollateralRatio`, `poolBorrowMax` and `effectiveScore` (followup task?)